src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  border: $boarder-color solid 1px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  height: 408;
+  width: 200px;
+  background-color: #fff;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 8px;
+    height: 134px;
+    width: 160px;
+    margin-left: 3px;
+  }
+
+  &__description {
+    margin-top: 32px;
+    margin-bottom: 4px;
+    color: $main-text-color;
+
+    font-family: "Roboto", sans-serif;
```

Comment:
looks redundant, cause you already added it in [page.scss](https://github.com/mate-academy/layout_product-cards/pull/3402/files#diff-8deb6ad9434036b981e3d6aef319f24873d90e6c492423380817519f22f8b0e5)